src/main/java/stack/StackArrayList.java

```diff
@@ -0,0 +1,41 @@
+package stack;
+
+public class StackArrayList {
+    int MAX_SIZE = 1000;
```

Comment:
we should not fix the size of Stack like this.
If you want to fix as default, then you should expend incase full
Or, you will delegate to initial Object StackArrayList by constructor. 
Ex:
 public StackArrayList(int size) {
     }
     public StackArrayList(int x, int size) {
         top = 0;
         this.A[top] = x;
     }